db-query-matchers.gemspec

```diff
@@ -15,10 +15,10 @@ Gem::Specification.new do |spec|
   spec.files         = Dir['lib/**/*.rb']
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'activesupport', '>= 4.0', "< 7"
+  spec.add_runtime_dependency 'activesupport', '>= 4.0'
```

Comment:
@nastya-soroko have you tested that it still works? It seems, that someone intentionally has excluded Rails 7, do you know why?

Reply:
@spieker it is very tricky to test without a package due to the gem source, it was ok so far,  but I won't merge before the rails 7.0.0  test is entirely done. and from https://github.com/suitepad-gmbh/db-query-matchers/commit/7e710821317e573db8b2ad6897feaaa7f5fbfaef it was just historically with upper range, I don't see any obvious reasons for this yet.